enterprise/server/auth/auth.go

```diff
@@ -458,20 +459,12 @@ func createAuthenticatorsFromConfig(ctx context.Context, env environment.Env, au
 }
 
 func newOpenIDAuthenticator(ctx context.Context, env environment.Env, oauthProviders []config.OauthProvider) (*OpenIDAuthenticator, error) {
-	oia := &OpenIDAuthenticator{
-		env: env,
-	}
-
-	myURL, err := url.Parse(env.GetConfigurator().GetAppBuildBuddyURL())
-	if err != nil {
-		return nil, err
-	}
-	authURL, err := myURL.Parse("/auth/")
-	if err != nil {
-		return nil, err
-	}
-	oia.myURL = myURL
-	oia.authenticators, err = createAuthenticatorsFromConfig(ctx, env, oauthProviders, authURL)
+	authenticators, err := createAuthenticatorsFromConfig(
+		ctx,
+		env,
+		oauthProviders,
+		build_buddy_url.BuildBuddyURL("/auth/"),
```

Comment:
golang discourages repeating the package name as part of package functions or structs since the caller has to repeat them and the repeated name does not convey anything new. Perhaps a name like `WithPath` would be better?